src/components/dropdown/daikin-dropdown.ts

```diff
@@ -467,6 +471,10 @@ export class DaikinDropdown extends LitElement {
     }
   }
 
+  /**
+   * This function expose to `daikin-input-group` and reflect it's attributes to `daikin-dropdown`.
+   * @ignore
```

Comment:
I think `@private` will work (tested on my machine). Can you confirm this?

```suggestion
   * @private
```